### ISSUE TITLE
How does the Drakvuf record the modification of the Windows Registry key value?

### ISSUE THREAD

**Issue by Skakf** [tklengyel/drakvuf#394]
Hello, Can I ask a question about the use of drakvuf?

I wonder if Drakvuf can record exactly which key in the Windows Registry has been changed and what the value has been changed to.
So, I did an experiment to modify the key value of the win7_VM registry while running Drakvuf on the host Ubuntu.

running: ./src/drakvuf -r /root/win7.rekall.json -d 1 

I modified the 'OEMBackground' Key value from 0 to 1 to change the background of login menu .
![image](https://user-images.githubusercontent.com/34418221/39514657-6bc3fa1e-4e2a-11e8-9731-b6946a144356.png)

After that, I get the data as following:
![image](https://user-images.githubusercontent.com/34418221/39515313-4cb932cc-4e2c-11e8-9d4b-cd7ce3d71847.png)

It seems that I could only know the prcoess 'regedit.exe' has been generated. As for the specified value, I couldn't get it.

So,  is there any way to find those detail data about the modification operation?

(I did this test  because maybe... it can help me understand how drakvuf record the operation of malware software step by step.)

Thank you very much!

**Comment by tklengyel** [comment]
Did you look for the output of the regmon plugin?

**Comment by Skakf** [comment]
Yeah, I found the data by filtering keyword 'REGMON' in vim.

![default](https://user-images.githubusercontent.com/34418221/39530711-1b199380-4e5c-11e8-8348-7c77c592cf54.png)

Thank you very much!

By the way, how can I get the output of the regmon plugin?
What options do I need to add after the ./src/drakvuf -r -d ?
I tried -S and -x， but also got the data that mixed syscall,poolmon and regmon together...
